ZConfig/cmdline.py

```diff
@@ -155,7 +157,7 @@ def get_section_info(self, type, name):
             bk = self.basic_key(s, pos)
             if name and self._normalize_case(s) == name:
                 L.append((optpath[1:], val, pos))
-            elif bk == type:
+            elif bk == type: # pragma: no cover
```

Comment:
Ugh, the shadowing of the `type` built-in is really ugly:  I spent more than a bit of time trying to work out how the code worked.  I wonder if we could rename the argument `type_`?